pixivpy3/aapi.py

```diff
@@ -58,28 +63,12 @@ def format_bool(self, bool_value):
     # 返回翻页用参数
     def parse_qs(self, next_url):
         if not next_url: return None
-        if sys.version_info >= (3, 0):
-            from urllib.parse import urlparse, unquote
-            safe_unquote = lambda s: unquote(s)
-        else:
-            from urlparse import urlparse, unquote
-            safe_unquote = lambda s: unquote(s.encode('utf8')).decode('utf8')
-
-        result_qs = {}
-        query = urlparse(next_url).query
-        for kv in query.split('&'):
-            # split than unquote() to k,v strings
-            k, v = map(safe_unquote, kv.split('='))
-
-            # merge seed_illust_ids[] liked PHP params to array
-            matched = re.match('(?P<key>[\w]*)\[(?P<idx>[\w]*)\]', k)
-            if matched:
-                mk = matched.group('key')
-                marray = result_qs.get(mk, [])
-                # keep the origin sequence, just ignore group('idx')
-                result_qs[mk] = marray + [v]
-            else:
-                result_qs[k] = v
+
+        query = up.urlparse(next_url).query
+        result_qs = up.parse_qs(query)
+        for key in result_qs:
+            if len(result_qs[key]) == 1:
+                result_qs[key] = result_qs[key][0]
```

Comment:
The returned params contains an array like `seed_illust_ids[]`, which needs to be parsed into a python array

Reply:
See [line 75](https://github.com/upbit/pixivpy/pull/124/files/f862895dde6e0f2ac991f6c6d56bb29910e4853c#diff-f603bba12ba259bd95cdcb4f29726d3fL75)

Reply:
Sorry for my negligence